libnavigation-core/src/main/java/com/mapbox/navigation/core/fasterroute/FasterRouteDetector.kt

```diff
@@ -16,7 +16,7 @@ internal class FasterRouteDetector(
         val alternativeDuration = alternativeRoute.duration()
         val weightedDuration = routeProgress.durationRemaining * PERCENTAGE_THRESHOLD
         val isRouteFaster = alternativeDuration < weightedDuration
-        return@withContext isRouteFaster && routeComparator.isRouteDescriptionDifferent(
+        return@withContext isRouteFaster && routeComparator.isSameRoute(
```

Comment:
It seems we inverted the logic but we didn't add `!routeComparator.isSameRoute` `!`. Did we miss it?

Reply:
😱  yeah

Reply:
wow thanks and nice catch!

negating and renaming broke the tests inside `FasterRouteDetectorTest`. so it wasn't caught from the unit tests i was relying on 😅 . all updated